service_container/tags.rst

```diff
@@ -214,11 +214,65 @@ method::
         }
     }
 
+To go even further, we can update our ``SensitiveElement`` attribute to be
+usable on methods::
+
+    // src/Attribute/SensitiveElement.php
+    namespace App\Attribute;
+
+    #[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD)]
+    class SensitiveElement
+    {
+        // ...
+    }
+
+We must now update the call to
+:method:`Symfony\\Component\\DependencyInjection\\ContainerBuilder::registerAttributeForAutoconfiguration`
+to support ``ReflectionMethod`` and ``ReflectionProperty``::
```

Comment:
Not ReflectionProperty here IIUC

Reply:
Artifact from my first draft, that's removed. Thanks!